clientlibrary/worker/worker.go

```diff
@@ -107,6 +109,13 @@ func (w *Worker) WithCheckpointer(checker chk.Checkpointer) *Worker {
 	return w
 }
 
+// WithLeasestealer is used to provide a custom leasestealer service for non-dynamodb implementation
```

Comment:
Leasetealer should be part of Checkpointer implementation. Similar to:
https://github.com/patrobinson/gokini/blob/master/consumer.go#L136

Reply:
Fixed